helpers/persistUnableToEnterDate.ts

```diff
@@ -0,0 +1,44 @@
+import { TransactionMode } from "remultiform/database";
+import ProcessDatabaseSchema from "storage/ProcessDatabaseSchema";
+import Storage from "../storage/Storage";
+import keyFromSlug from "./keyFromSlug";
+
+export enum FailedAttempts {
+  First = "firstFailedAttempt",
+  Second = "secondFailedAttempt",
+}
+
+export const persistUnableToEnterDate = async (
+  attemptKey: FailedAttempts
```

Comment:
```suggestion
  attemptKey: keyof ProcessDatabaseSchema["schema"]["unableToEnter"]["value"]
```

Reply:
Oh I see. We use these elsewhere. Leaving the comment as it might be helpful anyway.